scripts/linters/excluded_any_type_files.json

```diff
@@ -61,6 +61,7 @@
     "core/templates/domain/skill/MisconceptionObjectFactorySpec.ts",
     "core/templates/domain/skill/skill-creation-backend-api.service.ts",
     "core/templates/domain/skill/skill-mastery-backend-api.service.ts",
+    "core/templates/domain/skill/skill-rights-backend-api.service.ts",
```

Comment:
@bansalnitish So we can remove this now?

Reply:
Done!